projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -689,11 +689,16 @@ export class IgxTimePickerComponent extends PickerBaseDirective
 
     /** @hidden @internal */
     public validate(control: AbstractControl): ValidationErrors | null {
-        const value = control.value;
-        const errors = {};
-        if (!value) {
-            Object.assign(errors, { value: true });
+        if (!control.value) {
```

Comment:
Could the value be `""`? If yes, will the behavior be expected?

Reply:
Yes, I think this behavior is expected since if we have no value / we receive a falsy value that the component cannot work with, we don't want to immediately make the model invalid. Because this would mean that upon initialization with no value bound to the picker, the model will be initially invalid. This excludes `InvalidDate` which is not a falsy value but the model should become invalid regardless.